x-pack/test/functional/apps/discover/reporting.ts

```diff
@@ -69,14 +69,83 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await PageObjects.reporting.setTimepickerInDataRange();
         await PageObjects.discover.saveSearch('my search - with data - expectReportCanBeCreated');
         await PageObjects.reporting.openCsvReportingPanel();
-        expect(await PageObjects.reporting.canReportBeCreated()).to.be(true);
```

Comment:
Got it, so we were previously testing that reporting is available, but not the contents of the reports.